e2e_mysql_test.go

```diff
@@ -54,25 +57,52 @@ func requireMySQLVars(t *testing.T) {
 	}
 }
 
+type mockResolver struct {
+}
+
+func (r *mockResolver) Resolve(_ context.Context, name string) (instanceName instance.ConnName, err error) {
+	if name == "mysql.example.com" {
+		return instance.ParseConnNameWithDomainName(mysqlConnName, "mysql.example.com")
```

Comment:
is this what we would expect a global DNS format to look like? Or would it be in the form with the trailing dot similar to PSC?

Reply:
The domain name is customer-defined and customer-entered, so I'd expect it to omit the trailing dot.